network.env.dist

```diff
@@ -100,6 +100,11 @@ WS_ADDR=localhost
 WS_API=eth,quai
 HTTP_API=quai
 
+#Networking Variables
+MAX_PEERS=50
+ENABLE_NAT=false
```

Comment:
why is the position of enable nat changing?

Reply:
So that its included with networking variables, such as nat, and ip address. In the new force push, the network name is also moved there to be organized. Everything is pretty scattered now.

Reply:
cool makes sense